js/components/wizard_page_query.jsx

```diff
@@ -82,28 +75,55 @@ function Query() {
             onClickTopicButton={onClickTopicButton}
           />
         </Modal>
+        {selectedTopic ?
+          <SubmitButton
+            setSubmitted={setSubmitted}
+            submitRequest={actions.submitRequest}
+            query={query}
+            selectedTopic={selectedTopic}
+            submitted={submitted}
+          /> : null}
 
+        <Heading tag="h2">Or search for something else</Heading>
+        <WizardHtml mid="query_slide_2" />
         {exceededMaxLengthQuery && <p style={{ color: 'white' }}>Query is limited to 500 characters</p>}
-        {(query && query !== '' && !exceededMaxLengthQuery) || selectedTopic ? (
-          <div className="w-component-submit">
-            <Button
-              onClick={() => {
-                setSubmitted(true);
-                actions.submitRequest({
-                  query: query || '',
-                  topic: selectedTopic,
-                });
-              }}
-              disabled={submitted}
-            >
-              Submit
-            </Button>
-            {submitted && <WizardHtml mid="loading" />}
-          </div>
-        ) : null}
+        {(query && query !== '' && !exceededMaxLengthQuery) ?
+          <SubmitButton
+            setSubmitted={setSubmitted}
+            submitRequest={actions.submitRequest}
+            query={query}
+            selectedTopic={selectedTopic}
+            submitted={submitted}
+          /> : null}
+        <FormItem
+          type="textarea"
+          isLabelHidden
+          label="Query"
+          onChange={(e) => setQuery(e.target.value)}
+          value={query || ''}
+          disabled={Boolean(selectedTopic)}
+        />
       </Constrain>
     </PageTemplate>
   );
 }
 
+function SubmitButton(setSubmitted, submitRequest, query, selectedTopic, submitted) {
```

Comment:
This shouldn't work. Props is an object that has to be destructured:

```suggestion
function SubmitButton({ setSubmitted, submitRequest, query, selectedTopic, submitted }) {
```

Reply:
Ah yes!  Thank you!